packages/console/src/pages/SignInExperience/components/ManageLanguageModal/LanguageEditor.module.scss

```diff
@@ -7,20 +7,24 @@
     padding: _.unit(6) _.unit(5);
     font: var(--font-title-large);
     color: var(--color-text);
+    display: flex;
+    justify-content: space-between;
 
-    > span {
-      margin-left: _.unit(2);
-      font: var(--font-body-medium);
-      color: var(--color-caption);
-    }
+    .languageInfo {
```

Comment:
Don't you need to set `.languageInfo` to flex and use `align-items: center` to center align the language name and built-in tag?

Reply:
Cool, added.